runtime/src/test/java/org/corfudb/runtime/clients/CorfuLayoutHandlerTest.java

```diff
@@ -0,0 +1,263 @@
+package org.corfudb.runtime.clients;
+
+import io.netty.channel.ChannelHandlerContext;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.protocols.wireprotocol.LayoutPrepareResponse;
+import org.corfudb.runtime.exceptions.OutrankedException;
+import org.corfudb.runtime.exceptions.SerializerException;
+import org.corfudb.runtime.proto.service.CorfuMessage.HeaderMsg;
+import org.corfudb.runtime.proto.service.CorfuMessage.PriorityLevel;
+import org.corfudb.runtime.proto.service.CorfuMessage.ResponseMsg;
+import org.corfudb.runtime.view.Layout;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.corfudb.protocols.CorfuProtocolCommon.DEFAULT_UUID;
+import static org.corfudb.protocols.CorfuProtocolCommon.getUuidMsg;
+import static org.corfudb.protocols.service.CorfuProtocolLayout.getBootstrapLayoutResponseMsg;
+import static org.corfudb.protocols.service.CorfuProtocolLayout.getCommitLayoutResponseMsg;
+import static org.corfudb.protocols.service.CorfuProtocolLayout.getLayoutResponseMsg;
+import static org.corfudb.protocols.service.CorfuProtocolLayout.getPrepareLayoutResponseMsg;
+import static org.corfudb.protocols.service.CorfuProtocolLayout.getProposeLayoutResponseMsg;
+import static org.corfudb.protocols.service.CorfuProtocolMessage.getHeaderMsg;
+import static org.corfudb.protocols.service.CorfuProtocolMessage.getResponseMsg;
+import static org.corfudb.runtime.view.Layout.fromJSONString;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@Slf4j
+@RunWith(MockitoJUnitRunner.class)
+public class CorfuLayoutHandlerTest {
```

Comment:
Can you remove the tests in LayoutHandlerTest and replace them with these tests?

Reply:
How about just delete the old `LayoutHanlderTest`?